### ISSUE TITLE
Cannot call make_lookup without at least one filter defined

### ISSUE THREAD

**Issue by mfiers** [daler/biomartpy#2]
I was trying to get a list of all human genes by calling:

```
get_bm_lookup ('ensembl', 'hsapiens_gene_ensembl', 
    attributes=['ensembl_gene_id', 'external_gene_id'])
```

but it fails with a segfault (!):

```
Error: attempt to use zero-length variable name
Segmentation fault: 11
```

Which seems to be coming somewhere from the depths of rpy. If I call the same function with the filter:

```
{'biotype': ['protein_coding']}
```

it succeeds. 

Is this something that can be fixed?

Thanks, Mark

**Comment by daler** [comment]
Thanks for reporting this, now fixed in 804b3c6e4316ab9cd8047307a6f9e54bce699f64 with updated tests.   I've also uploaded the new version as v0.1.2 to PyPI.

**Comment by mfiers** [comment]
Thanks!
